elements/gnome-sdk-sdk/WebKitGTK.bst

```diff
@@ -0,0 +1,87 @@
+# Replaces sdk/WebKitGTK.bst from gnome-sdk.
```

Comment:
Ouch!

Do try hard to reuse the GNOME one, building this is going to waste so much time and computing power. :persevere: 

It seems the only difference is you're adding `-DENABLE_GLES2=ON` on ARM, GNOME might want to do the same?

Reply:
Yeah, that's all we're doing here, which is a bit sad. At the moment it's just blindly following the original [generate-manifest.py](https://github.com/endlessm/endless-sdk-flatpak/blob/master/generate-manifest.py). I'll do some archaeology and then ask around on IRC :)